src/System.Security.Cryptography.Cng/tests/Configurations.props

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netfx-Windows_NT;
```

Comment:
Those members aren't slated to be included in netstandard2.0. This seems to imply that in order to get an Cng netstandard configuration you will need to do some #ifdef'ing of that configuration around these APIs.